internal/model/rwfolder.go

```diff
@@ -874,16 +875,15 @@ func (p *rwFolder) shortcutFile(file protocol.FileInfo) (err error) {
 	t := time.Unix(file.Modified, 0)
 	err = os.Chtimes(realName, t, t)
 	if err != nil {
-		if p.lenientMtimes {
-			err = nil
-			// We accept the failure with a warning here and allow the sync to
-			// continue. We'll sync the new mtime back to the other devices later.
-			// If they have the same problem & setting, we might never get in
-			// sync.
-			l.Infof("Puller (folder %q, file %q): shortcut: %v (continuing anyway as requested)", p.folder, file.Name, err)
+		err = nil
+
+		// Try using virtual mtimes
+		info, errStat := os.Stat(realName)
```

Comment:
Thought about it, but will the `:=` syntax allow reusing a variable while declaring new ones?  (This isn't something I've tried previously in Go.)